### PR TITLE
refactor: Add breadcrumb component

### LINKED ISSUE #420: Refactor pages role judge and nav link

### PR DESCRIPTION
**Pull request by whDongRui**
* fix: Remove detail_bak file (not used)
* resolve #420

### CODE REVIEW

File: src/pages/Admin/Apps/Deploy/index.jsx [hunk 1]
```diff
@@ -293,7 +293,8 @@ export default class AppDeploy extends Component {
     const { isKubernetes } = appDeployStore;
     const { isLoading } = appStore;
     const title = `${t('Deploy')} ${appDetail.name}`;
-    const { isNormal } = user;
+    const { isNormal, isDev } = user;
+    const linkPath = isDev ? 'My Apps>Test>Deploy' : 'Store>All Apps>Deploy';
```
Comment (iwisunny) [inline src/pages/Admin/Apps/Deploy/index.jsx]: hard code link path ? `linkPath` is label or url ?

File: src/components/Layout/NavLink/index.jsx [hunk 2]
```diff
@@ -8,7 +8,9 @@ import { inject } from 'mobx-react/index';
 import styles from './index.scss';
 
 @translate()
-@inject('rootStore')
+@inject(({ rootStore }) => ({
+  user: rootStore.user
+}))
 @observer
 export default class NavLink extends Component {
```
Comment (iwisunny) [inline src/components/Layout/NavLink/index.jsx]: `NavLink`, the name is same with `react-router-dom`'s NavLink

File: src/components/Layout/NavLink/index.jsx [hunk 3]
```diff
@@ -21,12 +23,24 @@ export default class NavLink extends Component {
   };
 
   render() {
-    const { children, linkPath, t } = this.props;
+    const { children, linkPath, user, t } = this.props;
     const paths = linkPath.split('>');
     const linkLen = paths.length - 1;
 
     const pathToLink = {
```
Comment (iwisunny) [inline src/components/Layout/NavLink/index.jsx]: separate links to external config file

Comment (whDongRui) [reply src/components/Layout/NavLink/index.jsx]: ok. pathToLink define to  external config file.

File: src/pages/Admin/Apps/Deploy/index.jsx (hunk 1 above)
Comment (whDongRui) [reply src/pages/Admin/Apps/Deploy/index.jsx]: In page linkPath is string, in component change to corresponding link by pathToLink。

File: src/components/Layout/NavLink/index.jsx (hunk 2 above)
Comment (whDongRui) [reply src/components/Layout/NavLink/index.jsx]: ok.

File: src/pages/Admin/Apps/Deploy/index.jsx (hunk 1 above)
Comment (iwisunny) [reply src/pages/Admin/Apps/Deploy/index.jsx]: I think you can abstract your `NavLink` component to `Breadcrumb` component, and make it more reusable

**Pull request merged**